### PR TITLE
Translate custom directives  part within migration file to Spanish

### PR DESCRIPTION
**Pull request by FLarra**

### CODE REVIEW

File: src/v2/guide/migration.md [hunk 1]
```diff
@@ -684,45 +684,45 @@ You can use:
...
+Las directivas tienen un alcance de responsabilidad muy reducido: ahora solo se usan para aplicar manipulaciones del DOM de bajo nivel. En la mayoría de los casos, debería preferir usar componentes como la abstracción principal de reutilización de código.
 
-Some of the most notable differences include:
+Algunas de las diferencias más notables incluyen:
 
-- Directives no longer have instances. This means there's no more `this` inside directive hooks. Instead, they receive everything they might need as arguments. If you really must persist state across hooks, you can do so on `el`.
-- Options such as `acceptStatement`, `deep`, `priority`, etc have all been removed. To replace `twoWay` directives, see [this example](#Two-Way-Filters-replaced).
-- Some of the current hooks have different behavior and there are also a couple new hooks.
+- Las directivas ya no tienen instancias. Esto significa que no hay más `this` dentro de los ganchos de la directiva. En cambio, reciben todo lo que puedan necesitar como argumentos. Si realmente debe persistir el estado a través de los ganchos, puede hacerlo en `el`.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: I think that ``puede hacerlo en `el`.`` is a bit misleading. What do you think about specifying that `el` refers to the element?

File: src/v2/guide/migration.md [hunk 2]
```diff
@@ -684,45 +684,45 @@ You can use:
...
+- Opciones como `acceptStatement`, `deep`, `priority`, etc han sido removidas. Para reemplazar directivas `twoWay`, vea [este ejemple](#Two-Way-Filters-replaced).
+- Algunos de los ganchos actuales tienen un comportamiento diferente y también hay un par de ganchos nuevos.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: What do you think about:

```suggestion
- Algunos de los ganchos actuales tienen un comportamiento diferente y también hay algunos ganchos nuevos.
```

File: src/v2/guide/migration.md [hunk 3]
```diff
@@ -684,45 +684,45 @@ You can use:
...
 
-Fortunately, since the new directives are much simpler, you can master them more easily. Read the new [Custom Directives guide](custom-directive.html) to learn more.
+Afortunadamente, dado que las nuevas directivas son mucho más simples, puede dominarlas más fácilmente. Lea la nueva [Guía de directivas personalizadas](custom-directive.html) para obtener más información.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of defined directives. The helper will flag all of them, as it's likely in most cases that you'll want to refactor to a component.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos de directivas definidas. El asistente los marcará a todos, ya que es probable que en la mayoría de los casos desee refactorizar a un componente.</p>
```
Comment (Tintef) [inline src/v2/guide/migration.md]: `todos` should be `todas` as it's referring to "directivas", right?

```suggestion
  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos de directivas definidas. El asistente los marcará a todas, ya que es probable que en la mayoría de los casos desee refactorizar a un componente.</p>
```

File: src/v2/guide/migration.md (hunk 1 above)
Comment (sxcamacho) [reply src/v2/guide/migration.md]: puede hacerlo en el elemento `el` 
something like that right?

File: src/v2/guide/migration.md (hunk 2 above)
Comment (sxcamacho) [reply src/v2/guide/migration.md]: @FLarra @Tintef  In other places we are using `hooks` instead of `ganchos`. What do you think about use `hooks`?

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**